src/css/cs/items/_kitchensink.scss

```diff
@@ -278,6 +278,31 @@
     }
   }
 
+  // Custom margin-widths, PS does not deliver classes to allow different font-sizes
+  @for $i from 100 through 300 {
+    $width: $i * 1px;
```

Comment:
@christianv I was thinking that 1px increment gives us more options.  What are your concerns about it?  It makes the CSS too big?